CHANGELOG.md

```diff
@@ -3,11 +3,14 @@
 ### Features
 
 - `[@jest/environment, jest-runtime]` Allow passing a generic type argument to `jest.createMockFromModule<T>()` method ([#13202](https://github.com/facebook/jest/pull/13202))
+- `[expect]` Expose `ExpectationResult` type ([#13240])(https://github.com/facebook/jest/pull/13240))
+- `[jest-snapshot]` Expose `Context` type ([#13240])(https://github.com/facebook/jest/pull/13240))
```

Comment:
I used those in type tests, but in a way they can be useful to defined matcher functions (`ExpectationResult`) or to implement snapshot matchers (`Context`).